src/Intl/Grapheme/Grapheme.php

```diff
@@ -11,7 +11,11 @@
 
 namespace Symfony\Polyfill\Intl\Grapheme;
 
-\define('SYMFONY_GRAPHEME_CLUSTER_RX', \PCRE_VERSION >= '8.32' ? '\X' : Grapheme::GRAPHEME_CLUSTER_RX);
```

Comment:
is the float cast optimized at compile-time ? According to https://3v4l.org/HWo9A/vld vs https://3v4l.org/cs17n/vld, I would say no.

Reply:
I don't think compile time optims are enabled on 3v4l.org, see eg https://3v4l.org/3UbGb/vld

Reply:
hmm, indeed, it might depend whether those optimizations are done by the compiler itself or by OPCache (the first link I gave still shows an optimized condition for the JMPZ instead of performing a `IS_SMALLER_OR_EQUAL`)